front/app/resources/GetUsers.tsx

```diff
@@ -24,16 +24,25 @@ export type Sort =
   | '-role';
 
 export interface InputProps {
+  // Not used
   pageNumber?: number;
   pageSize?: number;
   sort?: Sort;
   search?: string;
+  // Not used
   groupId?: string;
+  // A project id is passed to this to only return users who are
+  // moderator for the project (includes admins)
   canModerateProject?: string;
+  // What is the use of this one?
   canModerate?: boolean;
+  // Doesn't work with false
   canAdmin?: boolean;
```

Comment:
Logging tickets to fix/review these two.

Reply:
Could we link those tickets in the TODO such that it easy to follow up for anyone else touching the code?